cpu/esp8266/include/common.h

```diff
@@ -83,6 +83,12 @@ extern "C" {
 #define CHECK_PARAM(cond)         if (!(cond)) return;
 #endif
 
+
+#define LOG_TAG_ERROR(tag, fmt, ...)   LOG_ERROR("[%s] " fmt, tag, ##__VA_ARGS__)
```

Comment:
unrelated? These are never used by `esp8266` code (yet).

Reply:
Not used in `cpu/esp8266` but in `cpu/esp_common/esp_now` netdev, e.g., here https://github.com/RIOT-OS/RIOT/blob/ac775b8ae5e3f5a385239a16ebbdcb505b1cae89/cpu/esp_common/esp-now/esp_now_netdev.c#L351

That is, when `esp_now` is used, they are required. These macros will be moved later to `cpu/esp_common` in further cleanup PRs. But for the moment and this PR, they are required in `cpu/esp8266`.